operator/option/config.go

```diff
@@ -201,6 +210,20 @@ const (
 	// Enabling this option reduces waste of IP addresses but may increase
 	// the number of API calls to AlibabaCloud ECS service.
 	AlibabaCloudReleaseExcessIPs = "alibaba-cloud-release-excess-ips"
+
+	// CiliumEndpointBatch options
+
+	// CEBUpsertSyncPeriod is the interval for reconciling newly created CEBs,
+	// modified CEBs, or CEBs with new cilium endpoints with the k8s-apiserver.
+	CEBUpsertSyncPeriod = "ceb-upsert-sync-period"
+
+	// CEBDeleteSyncPeriod is the interval for reconciling deleted CEBs
+	// or CEBs with only deleted cilium endpoints with the k8s-apiserver.
+	CEBDeleteSyncPeriod = "ceb-delete-sync-period"
+
+	// CEBMaxCepsInCeb is the maximum number of cilium endpoints allowed in single
+	// a CiliumEndpointBatch resource.
+	CEBMaxCepsInCeb = "ceb-max-ceps"
```

Comment:
s/ceb-mas-ceps/ceb-max-ceps-in-ceb/g

Reply:
I think is existing flag is simple and clear.

Reply:
It is not consistent with variable name, either change the flag or the variable name for easier search and code readability.

Reply:
Done.